### PR TITLE
add plugins install method go1.16

### LINKED ISSUE #682: Use go install over go get?
For the Go quick start docs (https://grpc.io/docs/languages/go/quickstart/) you'll find this cmd line snippet to install the binaries necessary to compile .proto into the gRPC Go code and the Protobuf Go code:

```
$ export GO111MODULE=on  # Enable module mode
$ go get google.golang.org/protobuf/cmd/protoc-gen-go \
         google.golang.org/grpc/cmd/protoc-gen-go-grpc
```

This documentation (https://developers.google.com/protocol-buffers/docs/reference/go-generated) recommends using `go install`. Also the 1.16 release notes section on modules (https://tip.golang.org/doc/go1.16#modules) seems to indicate that Go is moving away from `go get`. Definitely not an expert here, which is why this is an issue/question and not a PR. My sense is that it may depend on which version of Go you have installed. Maybe adding an additional block to the documentation and distinguishing between Go versions makes the most sense?

### PR DESCRIPTION
**Pull request by uta-mori**
Closes #682

### CODE REVIEW

**Comment by chalin** [comment]
We haven't yet resolved how #682 will be addressed, but in the meantime, can you sign the CLA? We can't accept the PR otherwise.

cc @dfawley

File: content/en/docs/languages/go/quickstart.md [hunk 1]
```diff
@@ -25,6 +25,15 @@ spelling: cSpell:ignore Fatalf GOPATH
       $ go get google.golang.org/protobuf/cmd/protoc-gen-go \
                google.golang.org/grpc/cmd/protoc-gen-go-grpc
       ```
+      
+      If you're using Go v1.16 or later:
```
Comment (dfawley) [inline content/en/docs/languages/go/quickstart.md:29]: Why not always use `go install`?

File: content/en/docs/languages/go/quickstart.md [hunk 2]
```diff
@@ -25,6 +25,15 @@ spelling: cSpell:ignore Fatalf GOPATH
...
+ 
+      ```sh
+      $ go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+      $ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```
Comment (dfawley) [inline content/en/docs/languages/go/quickstart.md:33]: Maybe we should prescribe a version # here (major+minor, not patch), so it doesn't change at all over time?

Comment (uta-mori) [reply content/en/docs/languages/go/quickstart.md:33]: You are correct. I'll change it as follows
```
	$ go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26.0
	$ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1.0
```

File: content/en/docs/languages/go/quickstart.md (hunk 1 above)
Comment (uta-mori) [reply content/en/docs/languages/go/quickstart.md]: I'll change it.

File: content/en/docs/languages/go/quickstart.md (hunk 2 above)
Comment (chalin) [reply content/en/docs/languages/go/quickstart.md]: @uta-mori: please drop the patch numbers:

- v1.26.0 --> v1.26
- v1.1.0 --> v1.1

**Comment by chalin** [comment]
@uta-mori: as I mentioned on March 15, you'll need to sign the CLA before this PR can be accepted.

File: content/en/docs/languages/go/quickstart.md (hunk 2 above)
Comment (chalin) [reply content/en/docs/languages/go/quickstart.md]: I've dropped the patch numbers.

**Review by dfawley — APPROVED** [review]
Yes, LGTM, thank you.

**Comment by chalin** [comment]
Thanks for the approval @dfawley

**Pull request merged**